tembo-stacks/src/apps/pganalyze.yaml

```diff
@@ -33,4 +33,4 @@ postgres_config:
   - name: auto_explain.log_nested_statements
     value: on
   - name: log_line_prefix
-    value: '%m [%p] %q[user=%u,app=%a] ',db=%d
+    value: '%m [%p] %q[user=%u,app=%a], db=%d'
```

Comment:
@ianstanton how should this configuration be formatted?

Reply:
The config value should look like `'%m [%p] %q[user=%u,app=%a] ',db=%d` , so maybe the yaml value should look like `"'%m [%p] %q[user=%u,app=%a] ',db=%d"`?